### PR TITLE
action to ensure the common folders are in sync

### PR DESCRIPTION
**Pull request by Jsyro**

### CODE REVIEW

File: .github/workflows/compare_common_folders.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    steps:
+      - uses: actions/checkout@v1
+      - name: compare-folders
+        run: |
+          diff -qr services/core-web/common services/minespace-web/common
+          if [ $? != 0 ]
+            then 
+            echo "'/services/core-web/common' does not match '/services/minespace-web/common'"
+            exit 7
```
Comment (pdittaro) [inline .github/workflows/compare_common_folders.yaml]: Should be 1

Comment (Jsyro) [reply .github/workflows/compare_common_folders.yaml]: you did this.

File: services/minespace-web/common/actionCreators/noticeOfWorkActionCreator.js [hunk 2]
```diff
@@ -136,7 +136,7 @@ export const fetchOriginalNoticeOfWorkApplication = (applicationGuid) => (dispat
 export const updateNoticeOfWorkApplication = (
   payload,
   nowApplicationGuid,
-  message = "Successfully updated Notice of Work"
+  message = "Successsfully updated Notice of Work"
```
Comment (exzizt) [inline services/minespace-web/common/actionCreators/noticeOfWorkActionCreator.js]: Successfully

File: services/core-web/common/actionCreators/noticeOfWorkActionCreator.js [hunk 3]
```diff
@@ -88,6 +88,10 @@ export const importNoticeOfWorkApplication = (applicationGuid, payload) => (disp
       createRequestHeader()
     )
     .then((response) => {
+      notification.success({
+        message: "Successfully verified the Notice of Works location",
```
Comment (exzizt) [inline services/core-web/common/actionCreators/noticeOfWorkActionCreator.js]: **Notice of Work's**

This function would be better if it took the message as a parameter.

**Pull request merged**